### ISSUE TITLE
Establish a self-mortgage threshold for validators to increase the cost of collusion among validators.

### ISSUE THREAD

**Issue by jiangfuyao** [paritytech/polkadot#2326]
I personally believe that the current network needs to be improved to  increase the cost of collusion among validators.

There is currently no own-stake threshold, multiple validators can be gradually initiated through high returns, recommended methods, and different fake identity. This situation needs further improvement.
Propose a few suggested options: 

1. Establish an own-stake threshold for validators. For example, if the validator own stake is less than 10% or 20% of the network `avg staked`, the validator will be chill.

2. If the validator own stake is less than 10% or 20% of `this validator total staked`, the validator will be chill. This kind of scheme needs to be considered. Suddenly a large amount of nominated validators will be accidentally or maliciously chilled.

3. Own stake amount is X, this validator maximum can be nominated 3*X.

4. Other smarter options.

**Comment by jiangfuyao** [comment]
Maybe it is better to transfer this issue to the Substrate repository?

**Comment by burdges** [comment]
All validators need roughly equal stake in polkadot, so reducing validator supply reduces scalability post parachain launch.  We want at least 10x our current validator supply eventually, so by then everything would change, and imposing self stake now slows us reaching that goal.

If we think validators need more direct skin-in-the-game then we'd actually set some minimum commission, or use some function of commission and self stake, or give nominators some self stake predicate, but..

We ultimately depend upon nominators choosing validators wisely, so they should nominate based upon personal connections.  We should tell them scary stories about nominating based purely upon rewards.

**Comment by Xa52c** [comment]
this is the most pressing issue with polkadot. the current state of the validator / nominator landscape is clearly broken. we have single entities like zug capital and p2p.org spinning up dozens of validators with millions of dots nominated, while putting up only 10 dots of their own stake! and legit validators such as myself, who have sizeable self-stakes cannot make it into the validator set because the minimum stake is currently 1.7M dot.

frankly i'm astounded that parity managed to produce such a sophisicated techincal product without managing to nail the fundamentals of the nothing at stake problem.

suggest you copy tezos' self-bond requirement / overdelegation system. that is an example of a healthy validator landscape with profitable commissions for validators.

**Comment by burdges** [comment]
We do want to avoid people self staking significantly more validators than for which they possess dots:  **Small time dot holder incur personal and systemic risks by nominating whales.**  

Is self stake well defined though?  Another party could stake your controller key, and organizations do this, or even certify your session keys with their controller. 

We could explore a "solo stake" notion however, meaning stake that nominates your validator and nobody else, meaning nominators who forego staking entirely if one particular looses. 

There exist several possible advantages "solo stake", like it avoids entwining economic realities with key management, and it encourages human relationships between validator operators and nominators.

p.s.  If you run multiple validators then you could self stake numerous validators with 10 dots each, and then have one large stash nominate them all, which sounds fine.  I'd agree 0.7% commission becomes quite suspicious though, but still the question remains: How many keys back them?  And do those keys back other validators?

**Comment by kianenigma** [comment]
I agree with the overall direction recommended by this issue and find it important to eventually move towards more restrictions on validators self-stake. 

In the meantime, I want to point out a detail that: 

> we have single entities like zug capital and p2p.org spinning up dozens of validators with millions of dots nominated, while putting up only 10 dots of their own stake! and legit validators such as myself, who have sizeable self-stakes cannot make it into the validator set because the minimum stake is currently 1.7M dot.

I’ve seen quite a lot as well, but what you need to assume is that it is quite likely that a large portion of the nomination is also coming from the same entity, potentially from an account without identity. I recall seeing accounts with very little self-stake, and commission 100%. To me this only means that they are nominating themselves, so it is technically not a nothing-at-stake situation, it is just disguised. 

That being said, there's no way to prove if a nomination is from the same validator entity or now, so we might as well try enforcing it.
